modules/ding_ting_frontend/ding_ting_frontend.install

```diff
@@ -1,22 +1,33 @@
 <?php
+/**
+ * @file
+ * Handle module installation and updates.
+ */
 
 /**
- * Enable ding_ekurser and ting_oembed modules
+ * Implements hook_uninstall().
  */
-function ding_ting_frontend_update_7001() {
-  module_enable(array('ding_ekurser', 'ting_oembed_features'));
+function ding_ting_frontend_uninstall() {
+  variable_del('ding_ting_frontend_group_holdings_available');
 }
 
 /**
  * Enable Ting infomedia module.
  */
 function ding_ting_frontend_update_7000(&$sandbox) {
-    module_enable(array('ting_infomedia'));
+  module_enable(array('ting_infomedia'));
 }
 
 /**
- * Implements hook_uninstall().
+ * Enable ding_ekurser and ting_oembed modules
  */
-function ding_ting_frontend_uninstall() {
-  variable_del('ding_ting_frontend_group_holdings_available');
+function ding_ting_frontend_update_7001() {
+  module_enable(array('ding_ekurser', 'ting_oembed_features'));
 }
+
+/**
+ * Enable Ting cover addi module.
+ */
+function ding_ting_frontend_update_7002(&$sandbox) {
+  module_enable(array('ting_covers_addi'));
```

Comment:
This assumes that all sites want to have it enabled. I only think this makes sense for sites which do not have a ting_covers implementation enabled already - e.g. eReolen.

I suggest that we check for hook implementations and enable based on that.